### ISSUE TITLE
Kickstart spec work on manifest.json

### ISSUE THREAD

**Issue by ariasuni** [w3c/webextensions#47]
The previous community group had [a section on manifest.json in their draft](https://browserext.github.io/browserext/#manifest-json). I propose that we use it as a base to discuss what we want/don’t want for our own draft, or even copy (some part of) it if we feel like it’s appropriate.

- Do we agree on the structure of this section?
- Do we want to start from that and adapt it?

For example, I’m not sure that the [5.4 The activeTab permission](https://browserext.github.io/browserext/#active-tab) section has exactly its place in this section. Maybe we would like to describe permissions in another section entirely, especially since the list of permissions varies a lot and depend on API namespace available on each platform.

Keep in mind that the goal is to have some minimal, useful text in the spec, to start iterating on it. :slightly_smiling_face:

**Comment by dotproto** [comment]
Thanks for kicking this off, @ariasuni. I agree that several subsections don't seem within the scope of the manifest file. 

Brief notes on the current text:

* 5.3 Permissions URL patterns - URL patterns are used outside the manifest file (e.g. event filters). This seems like it should either become a child of a more appropriate top level section or become its own top level section.
* 5.4 The activeTab permission - I agree with @ariasuni's assessment that this "Manifest Format" probably isn't the right place for this. I think we may want to have a top level section on permissions and have a dedicated subsection that explores this permission in more depth.
* 5.5 The required_keys key - I'm not familiar with this key and it's not clear to me why this was added as a direct child of the manifest format. I only spent a couple minutes Googling for information, but the only meaningful references I found were spect text itself.
* The BECG spec doesn't identify what JSON standard should be used to parse this file. There are several versions, but it appears that the current standards are [ECMA-404](https://www.ecma-international.org/publications-and-standards/standards/ecma-404/), [STD 90](https://www.rfc-editor.org/info/std90), and [ISO/IEC 21778:2017](https://www.iso.org/standard/71616.html). As best I can tell all 3 are compatible with each other. Since the spec texts notes that "browsers are expected to ignore '//' comments", it's worth noting that as of the 2019 revision, ECMA-262 is a strict superset of JSON. Unfortunately I could not find any well defined specifications that amount to "[existing JSON spec] + inline comments".

That's all I have time for at the moment. I'm planning to give this section a closer read and to follow up with other members of the Chrome team to discuss manifest properties in more detail.
